saleor/graphql/product/types/attributes.py

```diff
@@ -161,4 +161,9 @@ class Meta:
 
 class AttributeInput(graphene.InputObjectType):
     slug = graphene.String(required=True, description=AttributeDescriptions.SLUG)
-    value = graphene.String(required=True, description=AttributeValueDescriptions.SLUG)
+    value = graphene.String(
+        required=False, description=AttributeValueDescriptions.SLUG
```

Comment:
If this is deprecated we should add information in the description as well. See as it's done in other deprecated input fields.